tests/configs/user-config-test-driver-extension.h

```diff
@@ -0,0 +1,426 @@
+/**
```

Comment:
This file would be trivial to generate in terms of its content, and it's repetitive and error prone. With the overhaul of the libtestdriver1 generation, how hard would it be to fit its generation into the build scripts?

Reply:
Sure, I didn't mean to do it now, sorry for being unclear. Filed as https://github.com/Mbed-TLS/mbedtls-framework/issues/255